### ISSUE TITLE
Better Image Gallery tool?

### ISSUE THREAD

**Issue by bjornmeansbear** [OOKB/devoo-extras#6]
Is there somewhere I can just dump images and then have them show up on the site somehow?

**Comment by webmasterkai** [comment]
- Dropbox
- Backblaze B2
- Web GUI Upload to Firebase "Firestore"
- Web GUI Upload to graph.cool file upload service.

**Comment by bjornmeansbear** [comment]
This is obviously related to #5 & #4 

I either A) get a folder of images emailed or file dumped from the oil peeps and then I have to throw them somewhere, or b) the photographer they use for some fancier shots posts them to his flickr and I have to pull them down from there.

However, the images never have useful titles or descriptions or anything... so I want to easily get them up and visible (both on the site, and then in something for creating useful metadata)

We could start with just whatever works for us, and then if successful more towards making it so maybe I can get them to upload themselves? but maybe just focus on easy for us part

**Comment by webmasterkai** [comment]
I think we want everyone to be able to edit the meta information, upload new ones and whatever else.

**Comment by bjornmeansbear** [comment]
Yes. I'm just saying if we need to break this down into smaller chunks, everyone add/edit can be secondary to just something that allows for this period.

Also though, maybe that's where our faults lie, we end up stopping at the "we can sort of do this for this one piece of the project for launching" and then never get back to the everyone can do this part
